src/explore-education-statistics-admin/src/pages/release/pre-release/components/PreReleaseUserAccessForm.tsx

```diff
@@ -60,6 +60,10 @@ const PreReleaseUserAccessForm = ({
 
   const [invitePlan, setInvitePlan] = useState<PreReleaseInvitePlan>();
 
+  const isValidEmail = (input: string) => {
+    return /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i.test(input);
+  };
+
```

Comment:
This validation should allow apostrophes, eg. amy.o'benson@hiveit.co.uk (I have friends with the surname O'Neil who are always complaining about this!)

Reply:
Thanks, good point. Made a change to accept this and added the unit test `accepts a range of valid values without causing a validation error` to check the special chars and some other valid styles of address.